src/library/Connect/LocalEnvironmentManager.cs

```diff
@@ -468,6 +468,12 @@ private void _AddKubernetesServiceEnv(IDictionary<string, string> envVariables,
             envVariables[$"{serviceName}_SERVICE_PORT"] = port.ToString();
             envVariables[$"{serviceName}_PORT_{port}_{protocolUpper}_ADDR"] = host;
             envVariables[$"{serviceName}_SERVICE_HOST"] = host;
+
+            // if this is managed identity with useKubernetesServiceEnvironmentVariables set to true we have to update ms endpoint variable
+            // from dns name to ip:port 
+            if (this._useKubernetesServiceEnvironmentVariables && StringComparer.OrdinalIgnoreCase.Equals(serviceName, Common.Constants.ManagedIdentity.TargetServiceNameOnLocalMachine)) {
+                envVariables[ManagedIdentity.MSI_ENDPOINT_EnvironmentVariable] = "http://" + host + ":" + port + "/metadata/identity/oauth2/token";

```

Comment:
💡☕️ Is there anything like “https” for this context? If that’s the case might be enum.

Reply:
Nope, only http. To my knowledge there are no plans to make this or add https.